pkg/services/sqlstore/migrations/peakq_rename.go

```diff
@@ -0,0 +1,30 @@
+package migrations
+
+import "github.com/grafana/grafana/pkg/services/sqlstore/migrator"
+
+func AddPeakQRenameMigration(mg *migrator.Migrator) {
+	mg.AddMigration("Update values of apiVersion for querylibrary in resource", migrator.NewRawSQLMigration("").
+		SQLite("UPDATE resource SET \"value\"=REPLACE(\"value\", 'peakq.grafana.app/v0alpha1', 'querylibrary.grafana.app/v0alpha1') WHERE \"group\"='peakq.grafana.app';").
+		Postgres("UPDATE resource SET \"value\"=REPLACE(\"value\", 'peakq.grafana.app/v0alpha1', 'querylibrary.grafana.app/v0alpha1') WHERE \"group\"='peakq.grafana.app';").
+		Mysql("UPDATE resource SET \"value\"=REPLACE(\"value\", 'peakq.grafana.app/v0alpha1', 'querylibrary.grafana.app/v0alpha1') WHERE \"group\"='peakq.grafana.app';"))
+
+	mg.AddMigration("Update group values for querylibrary in resource", migrator.NewRawSQLMigration("").
+		SQLite("UPDATE resource SET \"group\"='querylibrary.grafana.app' WHERE \"group\"='peakq.grafana.app';").
+		Postgres("UPDATE resource SET \"group\"='querylibrary.grafana.app' WHERE \"group\"='peakq.grafana.app';").
+		Mysql("UPDATE resource SET \"group\"='querylibrary.grafana.app' WHERE \"group\"='peakq.grafana.app';"))
+
+	mg.AddMigration("Update group values for querylibrary in resource history", migrator.NewRawSQLMigration("").
+		SQLite("UPDATE resource_history SET \"group\"='querylibrary.grafana.app' WHERE \"group\"='peakq.grafana.app';").
+		Postgres("UPDATE resource_history SET \"group\"='querylibrary.grafana.app' WHERE \"group\"='peakq.grafana.app';").
+		Mysql("UPDATE resource_history SET \"group\"='querylibrary.grafana.app' WHERE \"group\"='peakq.grafana.app';"))
+
+	mg.AddMigration("Update values of apiVersion for querylibrary in resource_history", migrator.NewRawSQLMigration("").
+		SQLite("UPDATE resource_history SET \"value\"=REPLACE(\"value\", 'peakq.grafana.app/v0alpha1', 'querylibrary.grafana.app/v0alpha1') WHERE \"group\"='peakq.grafana.app';").
+		Postgres("UPDATE resource_history SET \"value\"=REPLACE(\"value\", 'peakq.grafana.app/v0alpha1', 'querylibrary.grafana.app/v0alpha1') WHERE \"group\"='peakq.grafana.app';").
+		Mysql("UPDATE resource_history SET \"value\"=REPLACE(\"value\", 'peakq.grafana.app/v0alpha1', 'querylibrary.grafana.app/v0alpha1') WHERE \"group\"='peakq.grafana.app';"))
+
+	mg.AddMigration("Delete peakq from resource_version", migrator.NewRawSQLMigration("").
```

Comment:
@ryantxu not sure what `resource_version` actually represents but it seams reasonable to just clean it up, right?

Reply:
cleaning it up seems fine -- it is used to make sure the resource_version is always increasing within a group